web/utils/types/post.ts

```diff
@@ -16,6 +16,7 @@ export enum Size {
 }
 
 export enum Breed {
+  None = "none", // None
```

Comment:
Remove `none` from here and the options - I think I talked to design about this in a leadership team meeting but they never got around to removing it. Sorry about that!